packages/core/src/DdSdkReactNative.tsx

```diff
@@ -212,6 +211,13 @@ export class DdSdkReactNative {
         return DdSdk.clearAllData();
     };
 
+    /**
+     * Returns `true` if the SDK was initialized, `false` otherwise.
+     */
+    public static wasInitialized() {
```

Comment:
it is better to use present tense here, this will also add the consistency with the naming on the iOS side

```suggestion
    public static isInitialized() {
```

backing field to be updated as well

Reply:
I thought the same, I will modify it.